deploy/kubernetes/resources/chirper/activity-stream-impl-statefulset.json

```diff
@@ -0,0 +1,99 @@
+{
+  "apiVersion": "apps/v1beta1",
+  "kind": "StatefulSet",
+  "metadata": {
+    "name": "activityservice"
+  },
+  "spec": {
+    "serviceName": "activityservice",
+    "replicas": 1,
+    "template": {
+      "metadata": {
+        "labels": {
+          "app": "activityservice"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "activityservice",
+            "image": "chirper/activity-stream-impl",
+            "imagePullPolicy": "Never",
+            "ports": [
+              {
+                "containerPort": 9000,
+                "name": "http-lagom-api"
+              },
+              {
+                "containerPort": 2551,
+                "name": "akka-remote"
+              }
+            ],
+            "env": [
+              {
+                "name": "CASSANDRA_SERVICE_NAME",
+                "value": "_native._tcp.cassandra.default.svc.cluster.local"
+              },
+              {
+                "name": "APPLICATION_SECRET",
+                "value": "activityservice-application-secret"
+              },
+              {
+                "name": "ACTIVITYSERVICE_BIND_PORT",
+                "value": "9000"
+              },
+              {
+                "name": "ACTIVITYSERVICE_BIND_IP",
+                "value": "0.0.0.0"
+              },
+              {
+                "name": "AKKA_ACTOR_SYSTEM_NAME",
+                "value": "activityservice-v1"
+              },
+              {
+                "name": "AKKA_REMOTING_BIND_PORT",
+                "value": "2551"
+              },
+              {
+                "name": "AKKA_REMOTING_BIND_HOST",
+                "value": "$HOSTNAME.activityservice.default.svc.cluster.local"
```

Comment:
https://github.com/kubernetes/kubernetes/issues/1331, envvars are not expanded inside another envvar definitions